cdp-agentkit-core/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 - Improved prompts for all actions.
 
+## [0.0.6] - 2024-12-03
+
+### Fixed
+
```

Comment:
version should be `0.0.8` for this release. Please add a description of what social features are added

Reply:
roger that!